presto-product-tests/conf/docker/multinode/compose.sh

```diff
@@ -2,6 +2,8 @@
 
 source ${BASH_SOURCE%/*}/../common/compose-commons.sh
 
+export TEMPTO_CONFIG_YAML=$( canonical_path ${TEMPTO_CONFIG_YAML:-"./conf/tempto/tempto-configuration-for-docker-default.yaml"} )
```

Comment:
We should use `${PRODUCT_TESTS_ROOT}` instead of `.` in the default value (hence the build failure https://travis-ci.org/Teradata/presto/jobs/163555938)